docs/uninstall-olm.md

```diff
@@ -0,0 +1,62 @@
+# How do I uninstall OLM?
+
+You can uninstall OLM in a symmetrical way as you did to install it. Specifically all OLM resources in [crds.yaml](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/upstream/quickstart/crds.yaml) and [olm.yaml](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/upstream/quickstart/olm.yaml) should be deleted.
+
+## Uninstall Released OLM
+
+For uninstalling release versions of OLM, you can use the following commands:
+
+```bash
+export OLM_RELEASE=<olm-release-version>
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${OLM_RELEASE}/crds.yaml
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${OLM_RELEASE}/olm.yaml
+```
+
+> NOTE: You can identify which version of OLM you are using by inspecting the version of the packageserver CSV.
+
+> ```bash
+> export OLM_NAMESPACE=<olm-namespace>
+> kubectl -n $OLM_NAMESPACE get csvs
+> NAME          DISPLAY        VERSION REPLACES PHASE
+> packageserver Package Server 0.13.0           Succeeded
+> ```
+
+## Uninstall From Git Repository Master Branch
+
+You can also uninstall OLM from the master branch of the [operator-framework/operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager/) repository with the following: 
+
+```bash
+kubectl delete -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/crds.yaml
+kubectl delete -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/olm.yaml
+```
+
+## Verify OLM Uninstall
+
+You can verify that OLM has been uninstalled successfully by making sure that OLM **owned** `CustomResourceDefinitions` are removed:
+
+```bash
+export OLM_NAMESPACE=<olm-namespace>
+kubectl get crd -n $OLM_NAMESPACE
+No resources found.
+```
+
+You can also check that the OLM `deployments` are terminated:
+
+```bash
+kubectl get deploy -n $OLM_NAMESPACE
```

Comment:
if the uninstall steps were followed, can't all of these steps just be "check that olm namespace is gone?"

Reply:
That was my first intention, but namespace can take a while to be deleted. I can just specify to check namespace as the main approach and have this as optional.